src/Shared/Recruit.Vacancies.Client/Application/Commands/CloseVacancyCommand.cs

```diff
@@ -0,0 +1,12 @@
+﻿using System;
+using Esfa.Recruit.Vacancies.Client.Domain.Entities;
+using Esfa.Recruit.Vacancies.Client.Domain.Messaging;
+using MediatR;
+
+namespace Esfa.Recruit.Vacancies.Client.Application.Commands
+{
+    public class CloseVacancyCommand : CommandBase, ICommand, IRequest
+    {
+        public Vacancy Vacancy { get; set; }
```

Comment:
Best to pass in the vacancy Id not the full vacancy.  We can't be sure when the command handler is going to run so best to get the handler to re-fetch the vacancy to ensure the window for concurrency issues is as small as possible

Reply:
👍

Reply:
👍